tests/js/spec/components/charts/eventsRequest.spec.jsx

```diff
@@ -517,4 +517,35 @@ describe('EventsRequest', function () {
       );
     });
   });
+
+  describe('timeframe', function () {
+    beforeEach(function () {
+      doEventsRequest.mockClear();
+    });
+
+    it('passes query timeframe start and end to the child if supplied by timeseriesData', async function () {
+      doEventsRequest.mockImplementation(() =>
+        Promise.resolve({
+          p95: {
+            data: [[new Date(), [COUNT_OBJ]]],
+            start: 1,
+            end: 2,
```

Comment:
start/end is declared to be a string above, can we either this a string, or correct the response to be a number?

Reply:
I've updated `start` and `end` to be `number` types now to support the epoch time change from the backend! Updated the numbers in this test to be a little more realistic as well.